pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/AvroSchema.java

```diff
@@ -128,8 +130,8 @@ public static void addLogicalTypeConversions(ReflectData reflectData, boolean js
             } catch (ClassNotFoundException e) {
                 // Skip if have not provide joda-time dependency.
             }
-            reflectData.addLogicalTypeConversion(new TimeConversions.TimestampMicrosConversion());
```

Comment:
Should move it 134 lines, otherwise we will not be able to parse `timestamp - micros` type when `jsr310ConversionEnabled == true`.